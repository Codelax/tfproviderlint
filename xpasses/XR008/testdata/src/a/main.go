package a

import (
	"context"
	"os/exec"
)

func f() {
	failing := exec.CommandContext // want "avoid os/exec.CommandContext"

	// Comment ignored

	//lintignore:XR008
	exec.CommandContext(context.Background(), "true")

	exec.CommandContext(context.Background(), "true") //lintignore:XR008

	// Failing

	exec.CommandContext(context.Background(), "true") // want "avoid os/exec.CommandContext"

	failing(context.Background(), "true")
}
