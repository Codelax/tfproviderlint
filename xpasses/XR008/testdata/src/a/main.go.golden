package a

import (
	"context"
	"os/exec"
)

func f() {
	failing := // want "avoid os/exec.CommandContext"

		// Comment ignored

		//lintignore:XR008
		exec.CommandContext(context.Background(), "true")

	exec.CommandContext(context.Background(), "true") //lintignore:XR008

	// Failing

	// want "avoid os/exec.CommandContext"

	failing(context.Background(), "true")
}
